lib/formatters/srt_formatter.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Subtitles.SrtFormatter do
+  def format(subtitles) do
+    (subtitles ++ [""])
+    |> Enum.with_index(1)
+    |> Enum.map(&build(&1))
+    |> Enum.join("\n\n")
+  end
+
+  defp build({"", _}), do: ""
+  defp build({sub, idx}) do
+    ["#{idx}\n#{format_time(sub.from)} --> #{format_time(sub.to)}"]
+    ++ (sub.parts |> Enum.map(& &1.text_data))
+    |> Enum.join("\n")
+  end
+
+  defp format_time(time), do: time |> Time.to_string() |> String.replace(".", ",")
```

Comment:
This is anguish... Total... pure anguish...

It seems like time has no formatting options for it's to_string function.

I would go and build this time formatter with string interpolation instead of relying on that Time.to_string() never changes.

Reply:
For sure, future proofing is always a good idea